src/components/country-data-card/country-data-card-styles.module.scss

```
@@ -0,0 +1,45 @@
+@import 'styles/ui.module';
+@import 'styles/settings';
+@import 'styles/typography-extends';
+@import 'styles/common-animations.module';
+
+.container {
+  @include backdropBlur();
+  display: flex;
+  opacity: 1;
+  flex-direction: column;
+  width: $sidebar-width;
+  overflow: hidden;
+  color: white;
```

In case we would like to make our white less white in the future 
```suggestion
  color: $white;
```